lib/sentinel/controllers/html/sessions_controller.ex

```diff
@@ -18,17 +18,16 @@ defmodule Sentinel.Controllers.Html.Sessions do
   Log in as an existing user.
   Parameter are "username" and "password".
   """
-  def create(conn, %{"username" => username, "password" => password}) do
+  def create(conn, %{"session" => %{"username" => username, "password" => password}}) do
     case Authenticator.authenticate_by_username(username, password) do
       {:ok, user} ->
         conn
         |> Guardian.Plug.sign_in(user)
         |> put_flash(:info, "Successfully logged in")
         |> redirect(to: "/")
-      {:error, _errors} ->
+      {:error, errors} ->
         conn
-        |> put_flash(:error, "Unable to authenticate successfully")
```

Comment:
I'm open to this change, but I'm not sure I understand why we'd want to replace this flash message with `errors.base`? Could you clarify that for me?